src/hyperion/experiment_plans/flyscan_xray_centre_plan.py

```diff
@@ -239,8 +238,13 @@ def run_gridscan_and_move(
     yield from setup_zebra_for_gridscan(fgs_composite.zebra)
 
     LOGGER.info("Starting grid scan")
-
+    yield from bps.stage(
+        fgs_composite.zocalo
+    )  # connect to zocalo and make sure the queue is clear
```

Comment:
Should: I'm a bit worried that we're not explicitly waiting on the queue being cleared. If we run this without specifying `wait` I think the behaviour looks to be that it will just not wait so we end up assuming the 2s queue clear is less than the first zocalo message we actually want. Probably a reasonable assumption but I would prefer if we put this in a group that we do wait on at some point, not sure when we would though?